src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -170,6 +171,7 @@ bool ngraph::pass::MOCTransformations::run_on_model(const std::shared_ptr<ngraph
     if (m_use_shapes) {
         common_fusions->add_matcher<ngraph::pass::NearestNeighborUpsamplingFusion>();
     }
+    common_fusions->add_matcher<ov::pass::GRUCellFusion>();
```

Comment:
performance measurements needed

Reply:
I think we can enable the transformation in MOC pipeline in the next PR
implementation can be merged in this PR

Reply:
done